edward/util.py

```diff
@@ -243,8 +299,16 @@ def logit(x):
     -------
     tf.Tensor
         size corresponding to size of input
+
+    Raises
+    ------
+    InvalidArgumentError
+        If the input is not between :math:`(0,1)` elementwise.
     """
-    x = tf.clip_by_value(x, 1e-8, 1.0 - 1e-8)
+    dependencies = [tf.assert_positive(x),
```

Comment:
This is an interesting decision, i.e., to not do any clipping. I think I agree with it.